runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/ConfigurationManager.java

```diff
@@ -208,10 +181,11 @@ public Void call() throws Exception
 
             namespace.id = supplyId.applyAsInt(namespace.name);
 
-            namespace.readURL = this::readURL;
+            namespace.readURL = this.readURL;
 
             // TODO: consider qualified name "namespace::name"
-            namespace.resolveId = name -> name != null ? NamespacedId.id(namespace.id, supplyId.applyAsInt(name)) : 0L;
+            final NamespaceConfig finalNamespace = namespace;
+            namespace.resolveId = name -> name != null ? NamespacedId.id(finalNamespace.id, supplyId.applyAsInt(name)) : 0L;
```

Comment:
Is the rationale for adding `finalNamespace` here to make sure that `namespace` is not changing before lambda is executed?

If so, then please use `namespace0` instead of `finalNamespace` as we use that convention elsewhere in the codebase.

Reply:
Yes, it's necessary, otherwise it's not building: "Variable used in lambda expression should be final or effectively final". Renamed it to namespace0.